src/epics_containers_cli/ioc/helm.py

```diff
@@ -124,13 +124,14 @@ def _install(
         # complicated stderr filter to suppress helm symlink warnings
         cmd = (
             f"bash -c "
-            f'"helm {helm_cmd} {self.ioc_name} {self.bl_chart_folder} '
+            f'"'
+            f"helm {helm_cmd} {self.ioc_name} {self.bl_chart_folder} "
             f"--version {self.version} --namespace {self.namespace} -f {values} "
             f"--set ioc_name={self.ioc_name} --set ioc_version={self.version} "
-            f" 2> >(grep -v 'found symbolic link' >&2)\""
+            f" 2> >(grep -v 'found symbolic link' >&2) "
+            f"{self.args}"
+            f'"'
         )
-        if self.args:
-            cmd += f" {self.args}"
```

Comment:
self.args is still in the wrong place. It needs to be adding args to the helm command, but at present is after the redirect.

Reply:
To test this run it against an actual IOC instance to see the yaml it would send to K8S - then deliberately break the yaml by changing beamline-chart/values.yaml. Now you should only see yaml if you use the debug flag.